codeclimate-radon

```diff
@@ -22,6 +32,5 @@ if os.path.exists("/config.json"):
                 python_paths.append(i)
         include_paths = python_paths
 
-if len(include_paths) > 0:
-    args = " ".join(include_paths)
-    os.system("radon cc {0} -n{1} --codeclimate".format(args, threshold))
+args = " ".join(include_paths)
+os.system("{0} cc {1} -n{2} --codeclimate".format(binstub, args, threshold))
```

Comment:
I think it'd be helpful to users to echo a message to `stderr` here saying which python version is being used. Is that reasonable to add?